src/components/SecondaryPanes/CommandBar.js

```diff
@@ -98,6 +97,21 @@ function debugBtn(onClick, type, className, tooltip, disabled = false) {
 }
 
 class CommandBar extends Component {
+  props: {
+    sources: Object,
+    selectedSource: Object,
```

Comment:
I think we can pull in types from the reducers here: `Source`, `SourceRecord`, or `SourcesMap`...